wradlib/io/dem.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# Copyright (c) 2011-2019, wradlib developers.
+# Distributed under the MIT License. See LICENSE.txt for more info.
+
+
+"""
+Digital elevation model I/O
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provide surface/terrain elevation information
+
+.. autosummary::
+   :nosignatures:
+   :toctree: generated/
+
+download_srtm
+get_srtm
+
+"""
+
+import numpy as np
+import os
+
+from osgeo import gdal
+import requests
+
+import wradlib.util
```

Comment:
please use relative import here and specify function: `from ..util import get_wradlib_data_path `

Reply:
OK for relative import.
But what is the rationale for specifying functions?
I got stuck several times with circular dependencies georef->io->hdf->zonalstats->georef
Not specifying functions solve this.
Is it not a general recommendation not to use this?
I think it is also easier to maintain

Reply:
I agree. The `from package import module` is only suggested for submodules not functions, see this module-doc [here](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package). 

And you are of course right, that is makes the code more self describing and maintainable.